src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1524,9 +1579,50 @@ def post_process(self, r):
                 while r["properties"]["provisioningState"].lower() == "inprogress":
                     r = self.client.show(self.cmd, self.get_argument_resource_group_name(), self.get_argument_name())
                     time.sleep(1)
-                linker_client.linker.begin_create_or_update(resource_uri=r["id"],
-                                                            parameters=item["parameters"],
-                                                            linker_name=item["linker_name"]).result()
+                service_connectors_def_list, service_bindings_def_list = parse_service_bindings(
+                    self.cmd, self.get_argument_service_bindings(), self.get_argument_resource_group_name(), self.get_argument_name())
+                self.set_argument_service_connectors_def_list(
+                    service_connectors_def_list)
+                if any(len(item["parameters"]) ==
+                       2 for item in service_connectors_def_list):
+                    updated_service_connectors = update_connectors_with_two_parameters(
+                        item, service_connectors_def_list.copy(), r["id"])
+                    check_bindings_and_raise_error(
+                        self.cmd,
+                        updated_service_connectors,
+                        service_bindings_def_list,
+                        self.get_argument_resource_group_name(),
+                        self.get_argument_name())
+                else:
+                    check_bindings_and_raise_error(
+                        self.cmd,
+                        service_connectors_def_list,
+                        service_bindings_def_list,
+                        self.get_argument_resource_group_name(),
+                        self.get_argument_name())
+
+                if len(item["parameters"]) == 1:
+                    linker_create_or_update(
```

Comment:
Should we check the result for `linker_create_or_update`?

Reply:
yess. will update!